client/analytics/report/orders/config.js

```diff
@@ -41,15 +43,12 @@ export const advancedFilterConfig = {
 		],
 		input: {
 			component: 'SelectControl',
-			options: [
-				{ value: 'pending', label: __( 'Pending', 'wc-admin' ) },
-				{ value: 'processing', label: __( 'Processing', 'wc-admin' ) },
-				{ value: 'on-hold', label: __( 'On Hold', 'wc-admin' ) },
-				{ value: 'completed', label: __( 'Completed', 'wc-admin' ) },
-				{ value: 'refunded', label: __( 'Refunded', 'wc-admin' ) },
-				{ value: 'cancelled', label: __( 'Cancelled', 'wc-admin' ) },
-				{ value: 'failed', label: __( 'Failed', 'wc-admin' ) },
-			],
+			getOptions: function() {
+				return apiFetch( { path: NAMESPACE + 'reports/orders/totals' } ).then( data =>
```

Comment:
Thanks, yeah I've been trying to think about the best place for this data to be fetched and stored. The endpoint is actually one used to render the orders report, `reports/orders/totals`, so it will likely be queried by the Orders page by the Store and accessible via selectors. My vote is to go this route here as well.

Reply:
FYI, we already have `wcSettings.orderStatuses` [populated on page load](https://github.com/woocommerce/wc-admin/blob/master/lib/client-assets.php#L87), I think it was added as part of #214

Reply:
😆 and of course I reviewed/approved that PR - no wonder the idea seemed good 🤦‍♂️

Reply:
😲 Whoops, I should have seen that! There's even an order-status component. It was a nice dive into Gutenberg Store at least. Thanks for the input